src/routes.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
For the routes I decided to use a simple file instead of a directory with individual files because we won't have many endpoints. But it's easy to change anyway :)

Reply:
acutally, I take that back and will make a directory :)